pennylane_lightning/lightning_gpu/_state_vector.py

```diff
@@ -238,19 +238,26 @@ def _apply_lightning_controlled(self, operation):
         Returns:
             None
         """
-        if self._mpi_handler.use_mpi:
-            raise DeviceError("Lightning-GPU-MPI does not support Controlled GlobalPhase gates.")
-
         state = self.state_vector
 
         basename = operation.base.name
+        method = getattr(state, f"{basename}", None)
         control_wires = list(operation.control_wires)
         control_values = operation.control_values
         target_wires = list(operation.target_wires)
-        # Apply GlobalPhase
-        inv = False
-        param = operation.parameters
-        state.apply(basename, control_wires, control_values, target_wires, inv, param)
+        if method is not None:  # apply n-controlled specialized gate
+            inv = False
+            param = operation.parameters
+            method(control_wires, control_values, target_wires, inv, param)
```

Comment:
Could you check if we can merge this section with the `device_base`, please? :rocket:

Reply:
Let's keep it as it is for now. We can revisit it once there are feature parity for all the backends